### PR TITLE
Reimplement toString() by eu.chargetime.ocpp.utilities.MoreObjects.to StringHelper.

### PR DESCRIPTION
**Pull request by sumlin**
Also add custom Calendar.toString() into StringHelper.

### CODE REVIEW

File: ocpp-v1_6/src/main/java/eu/chargetime/ocpp/model/core/AuthorizeConfirmation.java [hunk 1]
```diff
@@ -83,11 +83,12 @@ public int hashCode() {
         return Objects.hash(idTagInfo);
     }
 
-    @Override
-    public String toString() {
+//    @Override
+    public String toString2() {
```
Comment (TVolden) [inline ocpp-v1_6/src/main/java/eu/chargetime/ocpp/model/core/AuthorizeConfirmation.java]: toString2?

File: ocpp-v1_6/src/test/java/eu/chargetime/ocpp/model/test/StartTransactionRequestTest.java [hunk 2]
```diff
@@ -137,7 +137,7 @@ public void setTimestamp_calendarNow_timestampIsSet() {
 
         // When
         request.setTimestamp(now);
-
+        System.out.println(request.toString());
```
Comment (TVolden) [inline ocpp-v1_6/src/test/java/eu/chargetime/ocpp/model/test/StartTransactionRequestTest.java]: Why do you have a printline here?

File: ocpp-v1_6/src/main/java/eu/chargetime/ocpp/model/core/AuthorizeConfirmation.java (hunk 1 above)
Comment (sumlin) [reply ocpp-v1_6/src/main/java/eu/chargetime/ocpp/model/core/AuthorizeConfirmation.java]: Oh, sorry, a few experiments.

File: ocpp-v1_6/src/test/java/eu/chargetime/ocpp/model/test/StartTransactionRequestTest.java (hunk 2 above)
Comment (sumlin) [reply ocpp-v1_6/src/test/java/eu/chargetime/ocpp/model/test/StartTransactionRequestTest.java]: Meh, some debug too... Sorry.

**Pull request merged**